README.md

```diff
@@ -1,6 +1,34 @@
 # Zanata Docker Images
 
-Docker images for Zanata.
+Docker images for Zanata project.
+
+## Release docker image
```

Comment:
We need to run 5 commands (one build, tag for version and latest,  and push for version and latest). Using this script, I no longer forget to tag and push version and latest.